app/Domain/Integrations/FormRequests/UpdateIntegrationUrlsRequest.php

```diff
@@ -20,9 +21,33 @@ public function rules(): array
         ];
 
         return [
-            'loginUrl' => $urlValidation,
+            'integrationName' => ['required', 'string', 'max:255'],
+            'description' => ['required', 'string', 'max:255'],
+            'loginUrls.*' => Rule::forEach(fn () => $urlValidation),
             'callbackUrls.*' => Rule::forEach(fn () => $urlValidation),
             'logoutUrls.*' => Rule::forEach(fn () => $urlValidation),
+            'newIntegrationUrls.*' => Rule::forEach(fn () => [
```

Comment:
What kind of URLs are `newIntegrationUrls`?

Reply:
Those are urls that have been added in the frontend in the current usage of the website. 
The state for already existing urls that are being edited is in separate fields from the newly added urls.

Reply:
We better look into using upsert (update or insert depending on if it exists) logic